mplTemplate.vcxproj

```diff
@@ -55,107 +54,131 @@
+  <ImportGroup Label="PropertySheets" Condition="'$(Configuration)|$(Platform)'=='release|x64'">
     <Import Project="$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props" Condition="exists('$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props')" Label="LocalAppDataPlatform" />
   </ImportGroup>
   <PropertyGroup Label="UserMacros" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
-    <IntDir>x32_$(Configuration)\</IntDir>
-    <LinkIncremental>true</LinkIncremental>
-    <OutDir>x32_$(Configuration)\</OutDir>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='assert|win32'">
+    <IntDir>$(Configuration)32\</IntDir>
+    <LinkIncremental>false</LinkIncremental>
+    <OutDir>$(Configuration)32\</OutDir>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='assert|x64'">
+    <IntDir>$(Configuration)64\</IntDir>
+    <LinkIncremental>false</LinkIncremental>
+    <OutDir>$(Configuration)64\</OutDir>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">
-    <IntDir>x32_$(Configuration)\</IntDir>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='debug|win32'">
+    <IntDir>$(Configuration)32\</IntDir>
     <LinkIncremental>false</LinkIncremental>
-    <OutDir>x32_$(Configuration)\</OutDir>
+    <OutDir>$(Configuration)32\</OutDir>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
-    <IntDir>x64_$(Configuration)\</IntDir>
-    <LinkIncremental>true</LinkIncremental>
-    <OutDir>x64_$(Configuration)\</OutDir>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='debug|x64'">
+    <IntDir>$(Configuration)64\</IntDir>
+    <LinkIncremental>false</LinkIncremental>
+    <OutDir>$(Configuration)64\</OutDir>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='release|win32'">
+    <IntDir>$(Configuration)32\</IntDir>
+    <LinkIncremental>false</LinkIncremental>
+    <OutDir>$(Configuration)32\</OutDir>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
-    <IntDir>x64_$(Configuration)\</IntDir>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='release|x64'">
+    <IntDir>$(Configuration)64\</IntDir>
     <LinkIncremental>false</LinkIncremental>
-    <OutDir>x64_$(Configuration)\</OutDir>
+    <OutDir>$(Configuration)64\</OutDir>
   </PropertyGroup>
-  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
+  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='assert|win32'">
     <ClCompile>
-      <WarningLevel>Level3</WarningLevel>
-      <SDLCheck>true</SDLCheck>
-      <PreprocessorDefinitions>WIN32;_DEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
-      <ConformanceMode>true</ConformanceMode>
+      <PreprocessorDefinitions>%(PreprocessorDefinitions)</PreprocessorDefinitions>

```

Comment:
Shouldn't we directly specify <Optimization> with Disabled/MaxSpeed ​​to ensure our configuration approach is followed?